bayes_opt/bayesian_optimization.py

```diff
@@ -105,6 +76,27 @@ class BayesianOptimization(Observable):
         This behavior may be desired in high noise situations where repeatedly probing
         the same point will give different answers. In other situations, the acquisition
         may occasionally generate a duplicate point.
+
+    Attributes
```

Comment:
@adrianmolzon could you build the docs and check whether this renders correctly?